src/main/scala/nl.knaw.dans.easy.multideposit/package.scala

```diff
@@ -114,8 +114,8 @@ package object multideposit {
      */
     @throws[IOException]("in case of an I/O error")
     def writeXml(elem: Elem, encoding: Charset = encoding): Unit = {
-      Files.createDirectories(file.getParent)
-      XML.save(file.toString, elem, encoding.toString, xmlDecl = true)
+      Files.createDirectories(path.getParent)
+      XML.save(path.toString, XML.loadString(new PrettyPrinter(160, 2).format(Utility.trim(elem))), encoding.toString, xmlDecl = true)
```

Comment:
this was causing a bug in the formatting. To format the generated xml, you put it through the `PrettyPrinter`, but that returns a `String`, so we have to load it into `XML` again, such that the `XML.save` can call `toString` on it...